scripts/index.js

```diff
@@ -16,6 +16,26 @@ console.log(isEven(myNums));
 
 function squareNumbers(nums){
     let results = [];
-    nums.forEach();
+    nums.forEach(function(num){
+        results.push(num * num)
+    });
+    return results
 }
-console.log(squareNumbers(myNums));
\ No newline at end of file
+console.log(squareNumbers(myNums));
+
+const cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 }
+];
+
+function CitiesTemp(info,temp){
```

Comment:
It might be better to name this one something like `getCityTemps` so that it starts with a verb and a lowercase letter.